README.md

```diff
@@ -51,7 +57,7 @@ git clone https://github.com/ammen99/wf-recorder.git && cd wf-recorder
 meson build --prefix=/usr --buildtype=release
 ninja -C build
 ```
-Optionally configure with `-Ddefault_codec='codec'`. The default is libx264. Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`.
+Optionally configure with `-Ddefault_codec='codec'`, or modify meson_options.txt. The default codec is libx264. Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`.
```

Comment:
I don't see why modifying `meson_options.txt` is a good option at all.

Reply:
Frankly, I had no clue whatsoever what I should put in front of  `-Ddefault_codec=`, so I just fired up vim.....

Reply:
BTW you can run `meson configure` and it will print out a description of all build options and what their possible values are. :)